gensim/test/test_sklearn_api.py

```diff
@@ -1151,6 +1159,58 @@ def testModelNotFitted(self):
         self.assertRaises(NotFittedError, phrases_transformer.transform, phrases_sentences[0])
 
 
+class TestPhrasesTransformerCommonTerms(unittest.TestCase):
+    def setUp(self):
```

Comment:
Need to add a test with loading old `PhrasesTransformer` model with your new code

Reply:
I'm afraid I don't quite understand what you mean by that. 
There are other existing test cases that already test the `PhrasesTransformer` without the now new `common_terms` parameter. As long as they are not failing, shouldn't this be sufficient?
If this is not what you meant, please, try to explain to me exactly what you mean.

Reply:
Let me try

1. Create fresh venv
2. Install gensim from pip
3. Train `PhrasesTransformer` and save it
4. Add saved model to `gensim/test/test_data`
5. Switch to your current env, write a test when you load this `PhrasesTransformer` from previous steps and check that this works correctly (apply to some text and check output).